### ISSUE TITLE
Keyboard navigation broken on embedded / inline

### ISSUE THREAD

**Issue by 20v100** [eternicode/bootstrap-datepicker#1263]
On the demo page of the Bootstrap Datepicker, when selecting the embedded / inline mode, the keyboard navigation with the arrows does not work. It does not work either on the website I'm working on. I imagine someone would have reported the issue already but I could not find it in the this issue section.

**Comment by anurag1490** [comment]
I am also facing same issue. When i use startDate or data-attribute for this only down arrow works until you select a date and after selection it works but also accepts those date which you excluded initially.

**Comment by jelhan** [comment]
@20v100 I'm facing the same issue. Keyboard navigation seems to be supported only if there is an `input` having focus. So it's not supported at all for _embedded / inline_ mode and also not working if datepicker is opened other than giving focus to input (e.g. `show` method, component button).

@anurag1490 That was another bug. Should be fixed by #1687.

**Comment by newpen** [comment]
Is this fixed yet? The inline datepicker is still not focusable by keyboard even on the demo site.
https://uxsolutions.github.io/bootstrap-datepicker/?markup=embedded&format=&weekStart=&startDate=&endDate=&startView=0&minViewMode=0&maxViewMode=4&todayBtn=false&clearBtn=false&language=en&orientation=auto&multidate=&multidateSeparator=&keyboardNavigation=on&forceParse=on#sandbox
